src/data/frontend/profile/mappers.ts

```diff
@@ -82,15 +84,11 @@ export const mappedLocations = ['Poland', 'Europe', 'Other'].map(
 )
 
 // TODO: Szymon tu będzie działał
-export const mappedTechnologies = [
-  'Javascript',
-  'Python',
-  'Node.js',
-  'React.js',
-  'Vue.js',
-  'Angular',
-  'MongoDB',
-].map((technology) => ({
-  name: technology,
-  value: technology,
-}))
+export const mappedTechnologies: Promise<DropdownOption[]> = apiClient
```

Comment:
**issue:** mappers are only responsible for mapping data from one type to other type / format - not fetching it